src/enums/control.enum.ts

```diff
@@ -0,0 +1,35 @@
+export enum Control {

```

Comment:
Control is a little to generic and broad so let's name file control-technology.enum.ts and change enum to ControlTechnology